client/src/styles/components/_chat.scss

```diff
@@ -264,4 +263,46 @@
       }
     }
   }
+
+  .parent {
```

Comment:
Please give this a more descriptive name. I can't tell with one look what's this about